src/main/webapp/css/main.css

```diff
@@ -16,18 +16,104 @@
 
 /* Prevent the page from becoming too wide. */
 body {
+	background: #5433FF;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+  color: #828c9b;
+	font-family: Product Sans;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
   max-width: 600px;
+  width: 50%;
+
+}
+
+/* Creation of dropdown menu. */
+.dropbtn {
+  background-color: #828c9b;
+  border: none;
+ 	color: white;
+ 	cursor: pointer;	
+  font-size: 16px;
+	padding: 16px;
+  position: absolute;
+  right: 0;
+}
+
+.dropdown {
+  display: inline-block;
+  position: absolute;
+  right: 20px;
```

Comment:
what's the difference btwn using right and btwn using padding or margin?

Reply:
Right offsets the image, based on the side of the screen its on, whereas padding gives it space relative to where it is (If you have a white block, right makes the block x pixels away from the right of the screen, padding makes it seem like the image has a big transparent border around it)